examples/tests/invalid/cloud_function_expects_inflight.test.w

```diff
@@ -1,7 +1,7 @@
 bring cloud;
 
-new cloud.Function((name: str): str => {
-  return "Hello {name}";
+new cloud.Function((name: Json?): Json? => {
+  return "Hello {name?.tryAsStr() ?? "world"}!";
```

Comment:
Is this really needed? Why wouldn't this work?

```suggestion
  return "Hello {name ?? "world"}!";
```

Reply:
That will compile, but `name` (if it's non-nil) is `Json`, which means it will be stringified as a JSON -- with quotes. This way JSON round-tripping behavior behaves well:

```js
let json1 = Json { a: 123 };
let str1 = "{json1}"; // {"a":123}
let parsed1 = Json.parse(str1); // OK!

let json2 = Json "hello";
let str2 = "{json2}"; // "hello" (with the quotes!)
let parsed2 = Json.parse(str2); // OK!
```

(open ended question) Do you think if `x` is Json, then `"{x}"` should do something different than `Json.stringify(x)`?

Reply:
> (open ended question) Do you think if x is Json, then "{x}" should do something different than Json.stringify(x)?

Maybe... Pragmatically:

```js
let x: Json = "hello";

log("hello, {x}");
```

Would be nice if it printed:

```
hello, hello
```

Reply:
> (open ended question) Do you think if x is Json, then "{x}" should do something different than Json.stringify(x)?

Worth a dedicated thread IMO. I don't know what's the desired behavior off the top of my head 🤔 .

Reply:
I'll open a discussion. Let's address this when we're cleaning up the rest of the stringification system (toString methods etc.)